library/juniper_junos_config.py

```diff
@@ -395,6 +395,7 @@
         data is loaded line by line and may contain any configuration mode
         commands, such as set, delete, edit, or deactivate. This value must be
         specified if the new configuration is in set format.  
+      - B(patch) - Add the configuration to the existing configuration.
```

Comment:
Kindly update the doc in detail. This informations display in readme doc

https://www.juniper.net/documentation/en_US/junos/topics/topic-map/junos-config-files-loading.html

Reply:
done.